c++/src/kj/compat/http.c++

```diff
@@ -5394,6 +5421,12 @@ HttpClient::WebSocketResponse HttpClientErrorHandler::handleWebSocketProtocolErr
   };
 }
 
+kj::Exception WebSocketErrorHandler::handleWebSocketProtocolError(
+      WebSocket::ProtocolError protocolError) {
+  return KJ_EXCEPTION(FAILED, kj::str("code=", protocolError.statusCode,
+                                        ": ", protocolError.description));
```

Comment:
The explicit formatting of the error here actually confuses code which tries to parse KJ exception messages. It would be better to change this to:

```
  return KJ_EXCEPTION(FAILED, protocolError.statusCode, protocolError.description));
```

Reply:
Note that KJ will format this like:

> failed: WebSocket protocol error; protocolError.statusCode = 1009; protocolError.description = Message too large: 1000 > 900

Reply:
In Workers' sentry, this will mean all WebSocket protocol errors get grouped together but that's probably OK.